### PR TITLE
Add mdacli

### PR DESCRIPTION
**Pull request by PicoCentauri**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by PicoCentauri** [comment]
@conda-forge-admin, please ping team

File: recipes/mdacli/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+{% set sha256 = "a26ec097677937b66aab3330b17a6360aa96b6760295ae44792639158c35175d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/mdacli/meta.yaml:12]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: a26ec097677937b66aab3330b17a6360aa96b6760295ae44792639158c35175d
```

File: recipes/mdacli/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "mdacli" %}
+{% set version = "0.1.12" %}
+{% set sha256 = "a26ec097677937b66aab3330b17a6360aa96b6760295ae44792639158c35175d" %}
```
Comment (xhochy) [inline recipes/mdacli/meta.yaml:3]: Don't put the `sha256` into a variable. 
```suggestion
```

File: recipes/mdacli/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [python_impl == 'pypy']
+  skip: true  # [py<39]
+  skip: true  # [py>39]
```
Comment (xhochy) [inline recipes/mdacli/meta.yaml:18]: Don't skip PyPy by default, don't merge the PyPy migration PR later.
```suggestion
  skip: true  # [py<39]
  skip: true  # [py>39]
```

File: recipes/mdacli/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (xhochy) [inline recipes/mdacli/meta.yaml:22]:
```suggestion
```

**Comment by PicoCentauri** [comment]
Thanks for the review @xhochy!

**Pull request merged**